test/e2e_new/data/kubetest/kustomization/kustomizeversions.yaml

```diff
@@ -21,66 +21,66 @@ spec:
           set -o errexit
           [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
           GSUTIL=gsutil
-          if ! command -v $${GSUTIL} > /dev/null; then
+          if ! command -v ${GSUTIL} > /dev/null; then
```

Comment:
Why/how are we able to remove the escaping here?

Reply:
The escaping was used for envsubst, but clusterctl pre-envsubst doesn't respect $${ as an escape, so you'll love this, I had to add these variables to the e2e config variables:

```yaml
  # include hacks for a typical conformance bootstrap script
  GSUTIL: '${GSUTIL}'
  PACKAGE_VERSION: '${PACKAGE_VERSION}'
  SUDO: '${SUDO}'
  VERSION_REGEX: '${VERSION_REGEX}'
```